index.html

```diff
@@ -53,7 +53,12 @@ <h2>Software</h2>
             <td>.gif</td>
             <td>image/gif</td>
           </tr>
-        </tbody>
+            <tr>
```

Comment:
It's good to try to keep the indentation consistent, please. HTML is something of a nightmare to follow and tracing indents is your best bet for navigation.